docs/topics/auth/customizing.txt

```diff
@@ -832,7 +832,16 @@ Finally, the following forms are tied to
 :class:`~django.contrib.auth.models.User` and need to be rewritten or extended
 to work with a custom user model:
 
-* :class:`~django.contrib.auth.forms.UserCreationForm`
+* :class:`~django.contrib.auth.forms.UserCreationForm`: an extension of django.forms.ModelForm
```

Comment:
I wouldn't make these changes here. They are unrelated to the flow of this section.